### DISCUSSION TITLE
Discrepancy between `WA_WIKIPEDIA` and `VWA_WIKIPEDIA`

### DISCUSSION THREAD

**Discussion by xhluca** [ServiceNow/BrowserGym#300] [Q&A] [answered]
Right now, `WA_WIKIPEDIA` is set to this:

https://github.com/ServiceNow/BrowserGym/blob/a5e3b46eff38db2a727b67f808b5412298c049bd/browsergym/webarena/README.md?plain=1#L36

whereas `VWA_WIKIPEDIA` is set to this:

https://github.com/ServiceNow/BrowserGym/blob/a5e3b46eff38db2a727b67f808b5412298c049bd/browsergym/visualwebarena/README.md?plain=1#L37

Which one should be the correct base path? I feel it should be WA_WIKIPEDIA but I am not sure

**Comment by gasse** [comment, marked as answer]
They are both correct, the URL for WebArena has a trailing path, the one for VisualWebArena doesn't. This is in part why we had to define our own environment variables `WA_WIKIPEDIA` and `VWA_WIKIPEDIA` as `WIKIPEDIA` was overlapping.

**Reply by xhluca** [reply]
thanks. glad to know they are both valid
